packages/core/src/render/Frame.tsx

```diff
@@ -17,7 +17,7 @@ export const Frame: React.FC<Frame> = ({ children, json }) => {
 
   const [render, setRender] = useState<React.ReactElement | null>(null);
 
-  const initial = useRef({
+  const initialState = useRef({
```

Comment:
Is there a reason why `initialState` would be better? In terms of readability, I believe it's pretty self-explanatory already, and the addition of "state" to its name might be misleading in regards to React as this is not a state variable.

Reply:
I don't find `initial` to be readable. It's like if I had a `begin` or `end` variable. WHat does it do? this is effectively the `initialState` of the `Frame` so I think it's overall a better name for this variable.

Reply:
My only issue is that it's not actually a state value in `<Frame />`. What about renaming it to `initialProps` (or `initialPropsRef`) instead?

Reply:
sounds good